packages/circle-react-elements/src/components/hooks/useGetTransaction.ts

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState } from 'react';
+
+import { TransactionWithToken } from '~/lib/types';
+
+interface UseGetTransactionResult {
+  error: Error | undefined;
+  data: TransactionWithToken | undefined;
+  isLoading: boolean;
+  reFetch: () => Promise<boolean>;
+}
+
+export const useGetTransaction = (
```

Comment:
how does this hook work? it will only populate `data` when reFetch is called, but it doesn't fetch anything otherwise. this is not a standard pattern in react apps. it should populate data on the init too.

Reply:
remove hook